src/cowSdk.ts

```diff
@@ -0,0 +1,44 @@
+import { OrderBookApi } from '@cowprotocol/cow-sdk/order-book'
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+import { SubgraphApi } from '@cowprotocol/cow-sdk/subgraph'
+import { MetadataApi } from '@cowprotocol/cow-sdk/metadata'
+
+const orderBookSDKCache: Record<SupportedChainId, { prod: OrderBookApi | null; staging: OrderBookApi | null }> = {
+  [SupportedChainId.MAINNET]: { prod: null, staging: null },
+  [SupportedChainId.GNOSIS_CHAIN]: { prod: null, staging: null },
+  [SupportedChainId.GOERLI]: { prod: null, staging: null },
+}
+
+export function orderBookSDK(chainId: SupportedChainId, env: 'prod' | 'staging' = 'prod'): OrderBookApi {
```

Comment:
I like that now the SDK exports separate object for the OrderBookApi, etc. 

I think this function shouldn't have then the SDK in its name, and actually not be a function, and just the creation of the instance. Sure you could make a singleton function, but seems simpler to use if you export the instance directly.

So then you can import like

```ts
import { orderBookApi } from 'cowSdk'
```

Reply:
Done